website/docs/components/button/partials/code/how-to-use.md

```diff
@@ -30,7 +30,9 @@ By default, if you define an icon, it is placed in the leading position (before
 
 ### Icon-only Button
 
-!!! Info
+!!! Insight
+
+**Accessibility tips**
 
 To add a tooltip to an icon-only Button, here’s an example of how to do it in an accessible way: [Accessible Button Tooltip Pattern](https://codepen.io/melsumner/pen/bGGdmMV).
```

Comment:
Suggestion: this banner is irrelevant because there is a Tooltip component. We have plenty of examples of icon-only tooltips in the tooltip docs. I'd suggest removing the banner entirely.